android/src/com/mapswithme/maps/background/AbstractLogBroadcastReceiver.java

```diff
@@ -1,25 +1,25 @@
 package com.mapswithme.maps.background;
 
-import android.content.BroadcastReceiver;
 import android.content.Context;
 import android.content.Intent;
-import androidx.annotation.NonNull;
-
 import android.text.TextUtils;
 import android.util.Log;
 
+import androidx.annotation.NonNull;
+
+import com.mapswithme.maps.MwmBroadcastReceiver;
 import com.mapswithme.util.CrashlyticsUtils;
 import com.mapswithme.util.log.Logger;
 import com.mapswithme.util.log.LoggerFactory;
 
 import static com.mapswithme.maps.MwmApplication.backgroundTracker;
 
-public abstract class AbstractLogBroadcastReceiver extends BroadcastReceiver
+public abstract class AbstractLogBroadcastReceiver extends MwmBroadcastReceiver
```

Comment:
I wouldn't make all broadcast receivers descendants of MwmBroadcasReceiver, because not all needs in a heavyweight core. If you know specific case where is problem encountered I would prefer fix only that case. Could you provide the case in which the problem appears?

Reply:
Rollback some receivers